master/internal/usergroup/postgres_groups.go

```diff
@@ -367,6 +408,43 @@ func UpdateGroupAndMembers(
 	return users, newName, nil
 }
 
+// UpdateGroupsForMultipleUsers adds and removes group associations for multiple members.
+func UpdateGroupsForMultipleUsers(
+	ctx context.Context,
+	modUsers []model.UserID,
+	addGroups []int,
+	removeGroups []int,
+) error {
+	tx, err := db.Bun().BeginTx(ctx, nil)
```

Comment:
I know the other group APIs do this (I think it was from lack of knowledge of this) but can we use

```db.Bun().RunInTx(ctx, &sql.TxOptions{}, func(ctx context.Context, tx bun.Tx) error {```

personally think it just so much more error prone needing to defer and then we miss checking the rollback error

Reply:
✅